platform/core.windows/src/org/netbeans/core/windows/services/NbPresenter.java

```diff
@@ -1591,6 +1591,18 @@ private void initBounds() {
      */
     private Window findFocusedWindow() {
         Window w = KeyboardFocusManager.getCurrentKeyboardFocusManager().getFocusedWindow();
+        if( w == null ) {
+            // PR#5280
+            LOG.fine( () -> "No focused window, find mainWindow" );
```

Comment:
If there is no focused window, then try the mainWindow rather than centering on the JDK's default screen.